frontend/src/component/changeRequest/ChangeRequest/Changes/Change/NewEditChange.tsx

```diff
@@ -0,0 +1,232 @@
+    environment,
+    open,
+    onSubmit,
+    onClose,
+    featureId,
+}: IEditChangeProps) => {
+    const projectId = useRequiredPathParam('projectId');
+    const { editChange } = useChangeRequestApi();
+    const [tab, setTab] = useState(0);
+    const newStrategyConfiguration = useUiFlag('newStrategyConfiguration');
+
+    const [strategy, setStrategy] = useState<Partial<IFeatureStrategy>>(
+        change.payload,
+    );
+
+    const { segments: allSegments } = useSegments();
+    const strategySegments = (allSegments || []).filter((segment) => {
+        return change.payload.segments?.includes(segment.id);
+    });
+
+    const [segments, setSegments] = useState<ISegment[]>(strategySegments);
+
+    const strategyDefinition = {
+        parameters: change.payload.parameters,
+        name: change.payload.name,
+    };
+    const { setToastData, setToastApiError } = useToast();
+    const errors = useFormErrors();
+    const { uiConfig } = useUiConfig();
+    const { unleashUrl } = uiConfig;
+    const { isChangeRequestConfigured } = useChangeRequestsEnabled(projectId);
+
+    const { feature, refetchFeature } = useFeature(projectId, featureId);
+
+    const ref = useRef<IFeatureToggle>(feature);
+
+    const { data, staleDataNotification, forceRefreshCache } =
+        useCollaborateData<IFeatureToggle>(
+            {
+                unleashGetter: useFeature,
+                params: [projectId, featureId],
+                dataKey: 'feature',
+                refetchFunctionKey: 'refetchFeature',
+                options: {},
+            },
+            feature,
+            {
+                afterSubmitAction: refetchFeature,
+            },
+            comparisonModerator,
+        );
+
+    useEffect(() => {
+        if (ref.current.name === '' && feature.name) {
+            forceRefreshCache(feature);
+            ref.current = feature;
+        }
+    }, [feature]);
+
+    const payload = {
+        ...strategy,
+        segments: segments.map((segment) => segment.id),
+    };
+
+    const onInternalSubmit = async () => {
+        try {
+            await editChange(projectId, changeRequestId, change.id, {
+                action: strategy.id ? 'updateStrategy' : 'addStrategy',
+                feature: featureId,
+                payload,
+            });
+            onSubmit();
+            setToastData({
+                title: 'Change updated',
+                type: 'success',
+            });
+        } catch (error: unknown) {
+            setToastApiError(formatUnknownError(error));
+        }
+    };
+
+    if (!strategyDefinition) {
+        return null;
+    }
+
+    if (!data) return null;
+
+    return (
+        <SidebarModal
+            open={open}
+            onClose={onClose}
+            label='Edit change'
+            onClick={(e) => {
+                e.stopPropagation();
+            }}
+        >
+            <FormTemplate
+                modal
+                disablePadding
```

Comment:
We had to duplicate this component because we can not disablePadding on the form template without affecting the old design.